maintenance/Maintenance.php

```diff
@@ -132,6 +132,10 @@ public function __construct() {
 			? getenv( 'MW_INSTALL_PATH' )
 			: realpath( dirname( __FILE__ ) . '/..' );
 
+		# FANDOM change: add common MW directories to include_path to
+		# speed up possible require/include with relative paths.
+		ini_set( "include_path", "{$IP}:{$IP}/includes:{$IP}/languages:{$IP}/lib/vendor:.:" );
```

Comment:
`/lib/vendor` can highly likely go away as we now use Composer's autoloader for them.

Reply:
nah, it contains dependencies such as Solarium which are not managed by composer